website/src/Link.js

```diff
@@ -1,5 +1,8 @@
 import React, { PropTypes, Component } from 'react';
 
+const isModifiedEvent = (event) =>
```

Comment:
As I understand by modified event, it would mean an event got mutated/changed by some code